### PR TITLE
Feature/double digit folder structure

### PR DESCRIPTION
**Pull request by Domnom**

### CODE REVIEW

File: src/ctrls/PlaybookBuildCtrl.js [hunk 1]
```diff
@@ -29,6 +29,14 @@ class PlaybookBuildCtrl extends Controller{
             if(args[3] === 'usage') {
                 console.log(`If you run ${chalk.magenta('playbook build')} it will look for all the ${chalk.green('*.playbook.js')} files`);  
             }
+            else
+            {
+                global.playbook = require('../playbook.sdk').playbook;
+                global.step = require('../playbook.sdk').step;
+                PlaybookService.buildPlaybookJsonFromFolderPath(args[3])
```
Comment (mightyhorst) [inline src/ctrls/PlaybookBuildCtrl.js:36]: - [ ] todo - add task to backlog to change to *yargs* https://www.npmjs.com/package/yargs

File: src/models/playbook/PlaybookModel.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /**
  * @requires Services
  */
-import * as TextIndentService from '../../services/utils/TextIndentService';
+import * as TextService from '../../services/utils/TextService';
```
Comment (mightyhorst) [inline src/models/playbook/PlaybookModel.js:4]: more of a question - is this the syntax we need to use. 
eg. does `import {TextServiceThingy} from '../../services/utils/TextService';` not work ?

File: src/models/playbook/PlaybookModel.js [hunk 3]
```diff
@@ -41,6 +42,14 @@ export class PlaybookModel {
         return category;
     }
 
+    addStep(path, name)
+    {
+        this.stepModels.push({
```
Comment (mightyhorst) [inline src/models/playbook/PlaybookModel.js:47]: this is a better approach.

File: src/models/playbook/PlaybookModel.js [hunk 4]
```diff
@@ -50,10 +59,16 @@ export class PlaybookModel {
...
     {
-        let indent = TextIndentService.indent(indentSize);
+        let indent = TextService.indent(indentSize);
+
 
         let content = 'const path = require("path");\n\n';
-            content += 'playbook("' + this.name + '")\n';
+
+        this.stepModels.forEach((stepModel) => {
+            content += "const " + stepModel.name + " = require('./" + stepModel.path + "\');\n";
```
Comment (mightyhorst) [inline src/models/playbook/PlaybookModel.js:68]: I am very happy with this approach. 

A backlog task *could* be switched between builder pattern and imperative pattern based on size of commits 

I think this is unnecessary

**Review by mightyhorst — APPROVED** [review]
Approved - good work. Very thorough and man alot goes into the formatting doesnt it! 

I think we could look at handlebars in the future ...

File: src/models/playbook/PlaybookModel.js (hunk 2 above)
Comment (Domnom) [reply src/models/playbook/PlaybookModel.js:4]: I think import {} will work just fine, i just pull it all in so it feels like a class when its used

Comment (mightyhorst) [reply src/models/playbook/PlaybookModel.js:4]: ah gotcha - yeah cool good idea. I like that better

**Pull request merged**